README.aspnetcore.md

```diff
@@ -14,8 +14,8 @@ These images contain the runtime only. Use [`microsoft/aspnetcore-build-nightly`
 - [`1.1.6-jessie`, `1.1.6`, `1.1`, `1` (*1.1/jessie/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/jessie/runtime/Dockerfile)
 - [`2.0.5-stretch`, `2.0-stretch`, `2.0.5`, `2.0`, `2`, `latest` (*2.0/stretch/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.0/stretch/runtime/Dockerfile)
 - [`2.0.5-jessie`, `2.0-jessie`, `2-jessie` (*2.0/jessie/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.0/jessie/runtime/Dockerfile)
-- [`2.1.0-preview1-stretch`, `2.1.0-preview1` (*2.1/stretch/amd64/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.1/stretch/amd64/runtime/Dockerfile)
-- [`2.1.0-preview1-jessie` (*2.1/jessie/amd64/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.1/jessie/amd64/runtime/Dockerfile)
+- [`2.1.0-preview1-stretch-slim`, `2.1.0-preview1` (*2.1/stretch-slim/amd64/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/2.1/stretch-slim/amd64/runtime/Dockerfile)
```

Comment:
One thing I started doing in https://github.com/dotnet/dotnet-docker/pull/385 is to alphabetize the distros per version.  I think it helps the more distros that are added but it isn't as big of deal for ASP.NET Core yet because there are only two distros.

Reply:
Is that something we could easily add to the image builder? It looks like you are doing it manually via readmeOrder, right?

Reply:
The order of the entries in the manifest determines the build order.  The order is also used as the readme order if the readmeOrder property is not specified.  Functionality could be added to image-builder to have built in sort orders but it has been problematic in the past to make the logic abstract because of the "custom" sort orders we have had between the version numbers and image variants (e.g. runtime-deps, runtime, sdk).  We could take another look at this but for now, if you want to make this change you should be able to simply reorder the manifest slightly to achieve the desired behavior.

Reply:
Ok. Manually sorted for now.